extensions/resource-deployment/src/ui/resourceTypeWizard.ts

```diff
@@ -59,12 +73,19 @@ export class ResourceTypeWizard {
 		} else if (instanceOfAzureSQLDBDeploymentProvider(this.provider)) {
 			return new DeployAzureSQLDBWizardModel(this.provider, this);
 		}
-		// other types are undefined for now.
-		return undefined;
+		return new PageLessDeploymentModel(this.provider, this);
 	}
 
 	public async open(): Promise<void> {
-		this.model.initialize();
+		this.wizardObject = azdata.window.createWizard(this.resourceType.displayName, this.resourceType.name, 'wide');
```

Comment:
this.wizardObject and this.model should be set in the contructor

Reply:
It is here because when the user changes the model, I am recreating the wizard.

Reply:
Where is the user's chanage of model reflected on this object? Is it in set model()? if so the create this there. I would also recommend addig a comment that the wizardObject depends on model so recreate it.